src/Entity/Fileset.php

```diff
@@ -124,4 +124,21 @@ public function isDone() :bool
 
         return count($this->files->matching($criteria)) === 0;
     }
+
+    /**
+     * Returns the total filesize for this fileset.
+     *
+     * @return integer
+     */
+    public function getFileSize() :int
+    {
+        $fileSize = 0;
+        $filesArray = $this->files->toArray();
```

Comment:
I think we can remove this, if we are going to use foreach

Reply:
indeed not needed...test passes without it to verify

Reply:
I was lead to believe that I had to use an array, but ArrayCollection is iterable, I guess Collection is not. Fixed it.